src/snowflake/snowpark/modin/plugin/_internal/io_utils.py

```diff
@@ -7,6 +7,9 @@
 from collections.abc import Hashable
 from typing import Any, Callable, Union
 
+import numpy as np
+import pandas as native_pandas
```

Comment:
I think we use `native_pd`
```suggestion
import pandas as native_pd
```

Reply:
done